composer-dependency-analyser.php

```diff
@@ -9,5 +9,6 @@
     ->ignoreErrorsOnPackageAndPath('spatie/laravel-ignition', __DIR__.'/src/Location.php', [ErrorType::DEV_DEPENDENCY_IN_PROD])
     ->ignoreErrorsOnPackageAndPath('livewire/livewire', __DIR__.'/src/NightwatchServiceProvider.php', [ErrorType::DEV_DEPENDENCY_IN_PROD])
     ->ignoreErrorsOnPackageAndPath('livewire/livewire', __DIR__.'/src/Hooks/LivewireListener.php', [ErrorType::DEV_DEPENDENCY_IN_PROD])
+    ->ignoreErrorsOnPackageAndPath('aws/aws-sdk-php', __DIR__.'/tests/Feature/Sensors/JobAttemptSensorTest.php', [ErrorType::SHADOW_DEPENDENCY])
```

Comment:
If we are referencing the class in tests, it probably makes sense to include it in our dev dependencies instead of ignoring it.

Reply:
That makes sense!

Reply:
Done 1efd560609d4399809c6c0b9a4238093ee8be4f2